mithril-common/src/entities/signed_entity.rs

```diff
@@ -68,7 +70,7 @@ impl SignedEntity<CardanoTransactionsSnapshot> {
     cfg_test_tools! {
         /// Create a dummy [SignedEntity] for [CardanoTransactionsSnapshot] entity
         pub fn dummy() -> Self {
-            let block_number = 50;
+            let block_number = crate::entities::BlockNumber(50);
```

Comment:
Maybe import `BlockNumber` instead?

```
use crate::entities::{
    BlockNumber, CardanoTransactionsSnapshot, MithrilStakeDistribution, SignedEntityType, Snapshot,
};
```

```suggestion
            let block_number = BlockNumber(50);
```

Reply:
It's in a cfg_test_tools! scope so it's easier to do it like this